.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,7 @@
 - [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md)?
 - [ ] Did you make sure to update the docs?   
 - [ ] Did you write any new necessary tests?  
+- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](../CHANGELOG.md)?
```

Comment:
maybe full path as Will did for the CONTRUBUTION

Reply:
Done

Reply:
sure it can be extended later...